### ISSUE TITLE
[v3.0.0] TypeError: Failed to resolve module specifier "tslib"

### ISSUE THREAD

**Issue by heikkilamarko** [centrifugal/centrifuge-js#193]
Migrated my SvelteKit app to use `"centrifuge": "3.0.0"`. The Centrifugo server version is v4.

During runtime I get the following error message in the browser console:

```
Uncaught (in promise) TypeError: Failed to resolve module specifier "tslib". Relative references must start with either "/", "./", or "../".
```

Here is the error message from `Vite`:

<img width="1037" alt="Screenshot 2022-07-21 at 10 45 29" src="https://user-images.githubusercontent.com/3757314/180159206-70dfd837-5504-4d69-b767-93d11f6d4af5.png">

**Comment by FZambia** [comment]
Hello @heikkilamarko - thanks for report.

Any chance you can try changes in #194 to check whether it fixes an issue?

**Comment by heikkilamarko** [comment]
Thanks for the reply. I tried the changes and no longer see the `tslib` error message. Now I get the following error in browser console:

<img width="2048" alt="Screenshot 2022-07-21 at 13 30 46" src="https://user-images.githubusercontent.com/3757314/180194342-f23f894b-498f-4da3-8435-ead5fb7b22d0.png">
.

**Comment by heikkilamarko** [comment]
I tried replacing all `global` statements in `build/centrifuge.js`with `window`. After the replace things started to work okay.

**Comment by FZambia** [comment]
I pushed one more commit to use `globalThis` instead of `global`. `globalThis` should refer to `window` in a browser. Could you try again please?

**Comment by heikkilamarko** [comment]
I tried and now it's working okay. Thanks!

**Comment by FZambia** [comment]
Thanks! Just released 3.0.1 with these fixes.
